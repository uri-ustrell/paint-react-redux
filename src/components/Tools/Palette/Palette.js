import React from "react";

const Palette = () => <div className="paint-palette"></div>;

export default Palette;
