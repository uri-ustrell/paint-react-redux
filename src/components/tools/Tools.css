.paint-tools {
	grid-area: tools;
	display: flex;
	flex-flow: column nowrap;
	width: var(--tools-container-width);
}

.paint-tools > div {
	padding: var(--tool-padding);
	width: var(--tool-width);
	background-color: var(--tool-background-color);
}

.tool-btn {
	cursor: pointer;
}
