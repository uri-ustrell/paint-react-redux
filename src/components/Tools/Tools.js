import React from "react";
import Palette from "./palette/Palette";

const Tools = () => (
	<div className="paint-tools">
		<Palette />
	</div>
);

export default Tools;
