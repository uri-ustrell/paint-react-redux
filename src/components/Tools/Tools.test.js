import React from "react";
import { shallow } from "enzyme";
import Tools from "./Tools";
import Palette from "./palette/Palette";

describe("Tools Component", () => {
	let wrapper;

	beforeEach(() => (wrapper = shallow(<Tools />)));

	it("must exists", () => {
		expect(wrapper.find("div.paint-tools").length).toBe(1);
	});

	it("should render correctly", () => {
		expect(wrapper).toMatchSnapshot();
	});

	it("should render <Palette /> component", () => {
		expect(wrapper.containsMatchingElement(<Palette />)).toEqual(true);
	});
});
