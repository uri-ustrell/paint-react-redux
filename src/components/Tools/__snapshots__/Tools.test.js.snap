// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tools Component should render correctly 1`] = `
<div
  className="paint-tools"
>
  <Palette />
</div>
`;
